wagtailio/newsletter/templates/newsletter/newsletter_mjml.html

```diff
@@ -0,0 +1,96 @@
+{% load wagtailcore_tags wagtail_newsletter %}
+
+{% mrml %}
+    <mjml>
+        <mj-head>
+            <mj-attributes>
+                <mj-all font-family="Arial, 'Helvetica Neue', Helvetica, sans-serif" padding="0" />
+                <mj-section background-color="#fff" />
+                <mj-text font-size="17px" line-height="1.5" />
+            </mj-attributes>
+
+            <mj-style>
+                .rich-text p { margin-top: 1.5em }
+                .rich-text a { color: #007d7e }
+                h2 { margin: 0; padding: 0; line-height: 1.25; font-weight: bold; font-size: 31px }
+                .rich-text.rich-text--accent h2 { margin-bottom: 42px }
+                .rich-text.rich-text--accent a { color: #fff }
+            </mj-style>
+
+            {% if page.preview %}
+                <mj-preview>{{ page.preview }}</mj-preview>
+            {% endif %}
+        </mj-head>
+
+        <mj-body width="660px" background-color="#f4f4f4">
+            <mj-section padding="13px 0" background-color="">
+                <mj-column>
+                    <mj-text align="center" css-class="rich-text">
+                        <a href="{{ page.full_url }}">View this email online</a>
+                    </mj-text>
```

Comment:
Same as below - shouldn't this be hidden on the web version of the page?

Reply:
Fixed in bec1ad01.